translations/fr.yaml

```diff
@@ -442,9 +443,11 @@ general:
   validUsers: "utilisateurs valides"
   verificationPreviewFor: "Aperçu de vérification pour {name}"
   viewAll: "Voir Tous"
+  vocabulariesBySessionType: "Vocabulaires par Type de Seance"
   weak: Faible
   whichIs: "qui est"
   write: "Écriture"
+  xTermsFromVocabYusedWithSessionTypeZ: "{termsCount, plural, =1 {1 term} other {# termes}} du vocabulaire \"{vocabulary}\" {termsCount, plural, =1 {est appliqué} other {sont appliqués}} à {sessionsCount, plural, =1 {1 session} other {# sessions}} avec le type de session \"{sessionType}\"."
```

Comment:
Now that's what I call a name!

Reply:
Do we need some Spanish trans here?

Reply:
Spanish translations - courtesy of G00gle - are here: https://github.com/ilios/frontend/pull/7051/files#diff-647ea58af4907f7545a8d9536a12c7f695e0b0513eccdf84fcf17f511729f76dR449

and here 

https://github.com/ilios/frontend/pull/7051/files#diff-647ea58af4907f7545a8d9536a12c7f695e0b0513eccdf84fcf17f511729f76dR402